docs/network/architecture.md

```diff
@@ -75,20 +83,21 @@ The lifecycle of a typical operation under normal circumstances is as follows:
 1. A user submits a transaction to an EVM node.
 1. The EVM node forwards the transaction to the sequencer.
 1. The sequencer puts the transaction in its queue as soon as possible (less than 500ms after receiving it in a nominal scenario).
-1. The sequencer puts the transaction into a blueprint.
-1. The sequencer publishes the blueprint to the EVM nodes, which update their states based on the transactions in the blueprint.
-1. The sequencer publishes the blueprint to the Smart Rollup inbox on layer 1 via a Smart Rollup node running in operator or batcher mode.
-1. The Smart Rollup nodes tracking the state of Etherlink fetch the blueprint from the Smart Rollup inbox, read its transactions, and update their states.
-1. The EVM nodes check the state of the Smart Rollup nodes to verify that blueprints have successfully and faithfully been finalized on layer 1.
+1. The sequencer puts the transaction into a block.
```

Comment:
What do you mean by "as soon as it's applicable?" Does that just mean "as soon as it can?" Or is it possible to submit a transaction that is not "applicable" and the EVM node and/or sequencer ignores it?

Changed to:

1. A user submits a transaction to an EVM node.
1. The EVM node forwards the transaction to the sequencer.
1. The sequencer puts the transaction in its pool.
1. The sequencer puts the transaction into a block as soon as possible (less than 500ms after receiving it in a nominal scenario).

Reply:
Based on discussion:

1. A user submits a transaction to an EVM node.
1. The EVM node forwards the transaction to the sequencer when it is valid.
If users submit multiple transactions that depend on each other (that is, they have nonces that are not yet valid), the EVM node stores them until they are valid.
1. The sequencer puts the transaction in its pool.
1. The sequencer puts the transaction into a block as soon as possible (less than 500ms after receiving it in a nominal scenario).